support/faq/using-sink-server.md

```diff
@@ -5,12 +5,14 @@ description: "Sink testing allows you to send messages through Spark Post for in
 
 Sink testing allows you to send messages through SparkPost for integration testing without the messages being attempted for delivery to actual email addresses. Any messages using the sink server email address format will automatically fail as intended.
 
-To use the sink server, you will need to append the email addresses you are injecting into our system with `*.sink.sparkpostmail.com`. Please note, the asterisk (`*`) is a wildcard, where you will place the domain portion of your injected email address.
+To use the sink server, you will need to append the email addresses you are injecting into our system with `*.sink.sparkpostmail.com`. Please note, the asterisk (`*`) is a wildcard, where you will place the domain portion of your injected email address. You also need to prepend the campaign_id you are injecting into our system with `sparkpost-performance-test-*`. Please note, the asterisk (`*`) is a wildcard, where you will place anything as part of the campaign_id.
 
 ***Please note, any messages injected into our service, including those to the sink server, count against your plan's message allowance.**                                                                                                                               
 
 **Example**: 
 
 *Email Address* - john.doe@example.com
 
-*To test with sink server* - john.doe@example.com.sink.sparkpostmail.com
+*Campaign ID* - sparkpost-performance-test-1234
```

Comment:
I think this line is supposed to be the example of the original campaign id.  Like see above how `*Email Address*` is just the normal non-sink version.  So i think this should be 

```
*Email Address* - john.doe@example.com

*Campaign ID* - 1234

*To test with sink server* - set address to `john.doe@example.com.sink.sparkpostmail.com` and set campaign_id to `sparkpost-performance-test-1234`
```

Reply:
lol yep